docker-compose.yml

```
@@ -9,6 +9,8 @@ services:
       MINIO_ROOT_USER: minio
       MINIO_ROOT_PASSWORD: minio123
       MINIO_CI_CD: "1"
+      MINIO_NOTIFY_WEBHOOK_ENABLE_primary: "on"
+      MINIO_NOTIFY_WEBHOOK_ENDPOINT_primary: https://webhook.example.com
```

I'd just change here to all-uppercase (as in the example from https://min.io/docs/minio/linux/reference/minio-server/minio-server.html#webhook-service-for-bucket-notifications)
```suggestion
      MINIO_NOTIFY_WEBHOOK_ENABLE_PRIMARY: "on"
      MINIO_NOTIFY_WEBHOOK_ENDPOINT_PRIMARY: https://webhook.example.com
```

---

I initially had it that way, but even though it's not documented in that example it turns out that the resulting ARN is case-sensitive so when I used `MINIO_NOTIFY_WEBHOOK_ENABLE_PRIMARY` the ARN becomes `arn:minio:sqs::PRIMARY:webhook` instead of `arn:minio:sqs::primary:webhook`.

I'm fine with whichever you think makes the most sense, but since the configuration only needs to be updated once that seemed like an easier place to break from convention.

---

Ah, that's interesting! Thanks for clarifying! All good to merge, then!